content/bn/docs/concepts/scheduling-eviction/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "শিডিউলিং, প্রিএম্পশন এবং ইভিকশন"
+weight: 95
+content_type: concept
+description: >
+   কুবারনেটিসে, শিডিউলিং মানে হল নিশ্চিত করা যে পডগুলি নোডগুলির 
+   সাথে মিলিত হয়েছে কিনা যাতে কিউবলেট তাদের রান করতে পারে। 
+   প্রিএম্পশন হল স্বল্প অগ্রাধিকার পডগুলি বাতিল করার প্রক্রিয়া যাতে উচ্চ 
+   অগ্রাধিকার পডগুলি নোডগুলিতে শিডিউল করতে পারে। ইভিকশন হল 
+   রিসোর্স-ক্ষুধার্ত নোডগুলিতে এক বা একাধিক পডগুলি প্রত্যাহার করার প্রক্রিয়া।
+no_list: True
+---
+
+কুবারনেটিসে, শিডিউলিং মানে হল নিশ্চিত করা যে {{<glossary_tooltip text="পডগুলি" term_id="pod">}} 
+{{<glossary_tooltip text="নোডগুলির" term_id="node">}} সাথে মিলিত হয়েছে কিনা যাতে {{<glossary_tooltip text="কিউবলেট" term_id="kubelet">}} তাদের 
+রান করতে পারে। প্রিএম্পশন হল স্বল্প {{<glossary_tooltip text="অগ্রাধিকার" term_id="pod-priority">}} পডগুলি বাতিল করার প্রক্রিয়া
+যাতে উচ্চ অগ্রাধিকার পডগুলি নোডগুলিতে শিডিউল করতে পারে। ইভিকশন হল সম্পদ-ক্ষুধার্ত নোডগুলিতে এক বা 
+একাধিক পডগুলি প্রত্যাহার করার প্রক্রিয়া।
+
+## শিডিউলিং
```

Comment:
maintain proper line gaps

Reply:
![image](https://github.com/kubernetes/website/assets/78101731/e9a18284-619e-417c-a76b-9f61a5a78a3d)

Reply:
@itsPronay bhaiya fixed it now
![image](https://github.com/kubernetes/website/assets/111334392/e00b29b5-62f5-46e9-a0b1-f39ff0fe22d8)